packages/react/src/components/breadcrumb/Breadcrumb.stories.tsx

```diff
@@ -21,12 +23,37 @@ export default {
   },
 };
 
+const languages: LanguageOption[] = [
+  { label: 'Suomi', value: 'fi' },
+  { label: 'Svenska', value: 'sv' },
+  { label: 'English', value: 'en' },
+];
+
 export const Example = (args) => <Breadcrumb {...args} />;
 
 export const ExampleInHeader = (args) => {
   return (
     <Header>
       <Header.UniversalBar primaryLinkText="Helsingin kaupunki" primaryLinkHref="#" />
+      <Header.ActionBar
+        title="Helsingin kaupunki"
+        titleAriaLabel="Helsingin kaupunki"
+        titleUrl="https://hel.fi"
+        titleStyle={Header.TitleStyleType.normal}
+        logoAriaLabel="Service logo"
+        logoUrl="https://hel.fi"
+        menuButtonAriaLabel="Menu"
+      >
+        <Header.NavigationLanguageSelector languages={languages} ariaLabel="Kielen valinta">
+          <h3>Tietoa muilla kielillä</h3>
+          <Link external href="www.example.com">
+            Selkosuomi
+          </Link>
+          <Link external href="www.example.com">
+            Viittomakieli
+          </Link>
+        </Header.NavigationLanguageSelector>
+      </Header.ActionBar>
```

Comment:
The Breadcrumb's spacing doesn't correspond with Header.

Reply:
This is broken already in release-3.0.0 branch so we can make own ticket of this as well.